src/tests/ftest/dfuse/multi_user.py

```diff
@@ -0,0 +1,59 @@
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import general_utils
+
+from dfuse_test_base import DfuseTestBase
+
+
+class MultiUser(DfuseTestBase):
+    """Runs multi-user dfuse"""
+
+    def test_dfuse_mu_mount(self):
+        """This test simply starts a filesystem and checks file ownership
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm
+        :avocado: tags=dfuse
+        :avocado: tags=test_dfuse_mu_mount
+        """
+
+        self.add_pool(connect=False)
+        self.add_container(self.pool)
+
+        self.load_dfuse(self.hostlist_clients)
+
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+
+        root_dir = self.dfuse.mount_dir.value
+
+        ret = general_utils.run_pcmd(self.hostlist_clients,
+                                     'stat {}'.format(root_dir), expect_rc=0)
+        ret0 = ret[0]
+        print(ret0)
+        assert ret0['exit_status'] == 0
```

Comment:
If the hosts have different return codes, then `ret[0]` might not be sufficient. There's a newish run class for this. Also, the bare `assert` isn't recommended. You could use `self.assertTrue` or `self.assertEqual`. Something like:
```
        from run_utils import run_remote
        result = run_remote(self.log, self.hostlist_clients, 'stat {}'.format(root_dir))
        self.assertTrue(result.passed, 'Failed to stat dfuse root directory')
```